system-variables.md

```diff
@@ -3778,6 +3778,21 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - Range: `[100, 16384]`
 - This variable is used to set the maximum number of schema versions (the table IDs modified for corresponding versions) allowed to be cached. The value range is 100 ~ 16384.
 
+### tidb_max_dist_task_nodes <span class="version-mark">New in v9.0.0</span>
+
+- Scope:SESSION | GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Type: Integer
+- Default value: `-1`
+- Range: `-1` or `[1, 128]`
+- This variable defines the maximum number of TiDB nodes that can be used by the Distributed eXecution Framework (DXF). The default value is `-1`, which indicates automatic mode. In automatic mode, TiDB dynamically calculates the value as `min(3, tikv_nodes / 3)`, where `tikv_nodes` represents the number of TiKV nodes in the cluster.
+
+> **Note:**
+> 
+> If you explicitly set the [`tidb_service_scope`](#tidb_service_scope-new-in-v740) system variable for some TiDB nodes, the Distributed eXecution Framework will only schedule tasks to these nodes. In this case, even if `tidb_max_dist_task_nodes` is set to a larger value, the actual number of TiDB nodes used will not exceed the number of nodes explicitly set by `tidb_service_scope`.
+> For example, in a cluster with 10 TiDB nodes, if 4 nodes have the same `tidb_service_scope` value set to `group1`, and you also set `tidb_max_dist_task_nodes = 5`, the actual number of TiDB nodes participating in task execution will be `4`, instead of `5`.
```

Comment:
```suggestion
> For example, in a cluster with 10 TiDB nodes, if 4 nodes have the same `tidb_service_scope` value set to `group1`, and you also set `tidb_max_dist_task_nodes = 5`, the actual number of TiDB nodes participating in distributed task execution will be `4`, instead of `5`.
```